external-docs/docs/guides/inspect-api/example3.md

```diff
@@ -0,0 +1,196 @@
+
+## Loading the data
+
+The first step in the process is to load the CSV-W into csvcubed and create the Inspector object:
+
+```python
+>>> from csvcubed.inspect.inspectortable import Inspector
+
+>>> eurovision_inspector = Inspector('sweden-at-eurovision-complete-dataset.csv-metadata.json')
+```
+
+## Exploring the data
+
+You can then explore the structure of the data set:
+
+```python
+>>> eurovision.tables
+[DataCubeTable(csv_url='sweden-at-eurovision-complete-dataset.csv', title='Sweden at Eurovision Complete Dataset', shape=<CubeShape.Standard: 1>, data_set_uri='sweden-at-eurovision-complete-dataset.csv#dataset'),
+ CodeListTable(csv_url='entrant.csv', title='Entrant', concept_scheme_uri='entrant.csv#code-list'),
+ CodeListTable(csv_url='language.csv', title='Language', concept_scheme_uri='language.csv#code-list'),
+ CodeListTable(csv_url='song.csv', title='Song', concept_scheme_uri='song.csv#code-list'),
+ CodeListTable(csv_url='year.csv', title='Year', concept_scheme_uri='year.csv#code-list')]
+
+>>> primary_table = eurovision_inspector.tables[0]
+>>> primary_table
+DataCubeTable(csv_url='sweden-at-eurovision-complete-dataset.csv', title='Sweden at Eurovision Complete Dataset', shape=<CubeShape.Standard: 1>, data_set_uri='sweden-at-eurovision-complete-dataset.csv#dataset')
+
+>>> csv_url = primary_table.csv_url
+>>> csv_url
+'sweden-at-eurovision-complete-dataset.csv'
+
+>>> primary_table_columns = primary_table.columns
+>>> primary_table_columns
+OrderedDict([('Year',
+              DimensionColumn(dimension=LocalDimension(dimension_uri='sweden-at-eurovision-complete-dataset.csv#dimension/year', label='Year'))),
+             ('Entrant',
+              DimensionColumn(dimension=LocalDimension(dimension_uri='sweden-at-eurovision-complete-dataset.csv#dimension/entrant', label='Entrant'))),
+             ('Song',
+              DimensionColumn(dimension=LocalDimension(dimension_uri='sweden-at-eurovision-complete-dataset.csv#dimension/song', label='Song'))),
+             ('Language',
+              DimensionColumn(dimension=LocalDimension(dimension_uri='sweden-at-eurovision-complete-dataset.csv#dimension/language', label='Language'))),
+             ('Value',
+              StandardShapeObservationsColumn(unit=UnitsColumn(), measures_column=MeasuresColumn())),
+             ('Measure',
+              MeasuresColumn()),
+             ('Unit',
+              UnitsColumn()),
+             ('Marker',
+              AttributeColumn(attribute=LocalAttribute(attribute_uri='sweden-at-eurovision-complete-dataset.csv#attribute/observation-status', label='Observation Status'), required=False))])
+```
+
+## Formatting the data
+
+You can also load the CSV as a pandas DataFrame:
+
+```python
+>>> data_cube_repository = primary_table.data_cube_repository
+>>> df, errors = data_cube_repository.get_dataframe(csv_url=csv_url, include_suppressed_cols=False, dereference_uris=False)
```

Comment:
But it looks like the URIs have been dereferenced to their labels in this example?